### ISSUE TITLE
Internal test of TESS LCV - bug reporting

### ISSUE THREAD

**Issue by trouille** [zooniverse/front-end-monorepo#717]
On macbook pro with Chrome v73

I got this error after clicking 'done' on feedback modal after getting this gold subject incorrect: 

![Screen Shot 2019-04-12 at 11 32 23 AM](https://user-images.githubusercontent.com/4975655/56052387-e38f6500-5d16-11e9-8e40-323574a7d401.png)

**Comment by srallen** [comment]
This is expected since the new app doesn't have a page for Talk. Once deployed to production and we have the proxy redirects setup, it'll route to the old app.

**Comment by trouille** [comment]
Zoom worked on the first subject I classified, but when I went to use Zoom on the next subject, it didn't work. I refreshed the page and zoom worked again.

**Comment by srallen** [comment]
Zoom bug is throwing: 

`Uncaught DOMException: Failed to read the 'value' property from 'SVGLength': Could not resolve relative length`

**Comment by beckyrother** [comment]
In Chrome, I'm able to double-click to zoom in if I have the move tool active, but I can't zoom out or use the zoom in or out tools. 
Firefox works just fine.

**Comment by rogerhutchings** [comment]
@beckyrother I couldn't replicate your bug, unfortunately. In Chrome, I can:

- use the zoom in / out buttons when the annotate tool is active
- use the zoom in / out buttons when the move tool is active
- double-click to zoom when the move tool is active

**Comment by eatyourgreens** [comment]
I can reproduce @beckyrother's bug in Chrome if I move onto a second subject. The zoom in/out buttons stop working after the first subject. I get the same bug in Firefox, though, and it's also the bug mentioned in earlier comments. Full stack trace is:
```
Uncaught DOMException: Failed to read the 'value' property from 'SVGLength': Could not resolve relative length.
    at SVGRectElement.defaultExtent (webpack://@zooniverse/classifier/./node_modules/d3-zoom/src/zoom.js?:30:25)
    at new Gesture (webpack://@zooniverse/classifier/./node_modules/d3-zoom/src/zoom.js?:199:26)
    at gesture (webpack://@zooniverse/classifier/./node_modules/d3-zoom/src/zoom.js?:191:12)
    at SVGRectElement.eval (webpack://@zooniverse/classifier/./node_modules/d3-zoom/src/zoom.js?:165:40)
    at Dispatch.call (webpack://@zooniverse/classifier/./node_modules/d3-dispatch/src/dispatch.js?:58:72)
    at start (webpack://@zooniverse/classifier/./node_modules/d3-transition/src/transition/schedule.js?:129:13)
    at schedule (webpack://@zooniverse/classifier/./node_modules/d3-transition/src/transition/schedule.js?:79:32)
    at timerFlush (webpack://@zooniverse/classifier/./node_modules/d3-timer/src/timer.js?:66:48)
    at wake (webpack://@zooniverse/classifier/./node_modules/d3-timer/src/timer.js?:76:5)
```

**Comment by eatyourgreens** [comment]
Does zoom need to be reset between subjects? The code's a bit hard to follow.

**Comment by rogerhutchings** [comment]
I think `this.xAxis` and `this.yAxis` actually need to be redefined - I've chased the bug to https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js#L611

**Comment by eatyourgreens** [comment]
#720 fixes it by creating a fresh, new viewer for each subject.

**Comment by rogerhutchings** [comment]
Okay, I've tested this in Chrome and Firefox following @eatyourgreens' fix, and everything looks good.
